keyboards/lm_keyboard/lm60n/lm60n.h

```diff
@@ -166,8 +166,8 @@
     k40, k41, k42,                k43,                k44, k45, k46, k47,           k4c,      k4d       \
 ) { \
     { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e }, \
-    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, XXX, XXX }, \
-    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, XXX }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k2c, XXX }, \
```

Comment:
The naming is based off of row and column.  This shouldn't be changed here.  It should be changed in the top part